### PR TITLE
Moved umbrella chart docs into helm folder.

### PR DESCRIPTION
**Pull request by jacobcrawford**
Fixed path for docs and made reference from main readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Wiki
 
 Documentation related to git can be found in [git folder](./git/index.md)
+
+Documentation related to helm can be found in [helm folder](./helm/index.md)
```
Comment (skovbakke) [inline README.md:5]: I think we should move the index to a single index in the README file.

**Review by skovbakke — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (skovbakke) [reply README.md]: Nicely done.

**Review by skovbakke — APPROVED** [review]
Looks good

**Review by skovbakke — APPROVED** [review]
Very nice

**Review by jaxels10 — APPROVED** [review]
Ok.

**Pull request merged**